app/Models/Post.php

```diff
@@ -12,8 +12,13 @@ class Post extends Model
         'title',
         'description',
         'status',
+        'category',
         'created_at',
         'updated_at',
         'deleted_at'
     ];
+
+    public function categories() {
+        return $this->belongsToMany(Category::class,'category_posts','post_id','category_id')->withTimestamps();
+    }
```

Comment:
_If your pivot table name is **category_post** you can shorten it like the following sentence_

`return $this->belongsToMany(Category::class);`